jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/ClickAwayListenerPage.java

```diff
@@ -0,0 +1,37 @@
+package io.github.com.pages.utils;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.TextArea;
+
+public class ClickAwayListenerPage extends WebPage {
+
+    @UI("//h2[text()='Example']/following-sibling::div[1]/div/button")
+    public Button exampleButton;
+
+    @UI("//h2[text()='Portal']/following-sibling::div[1]/div/button")
+    public Button portalButton;
+
+    @UI("//div[text()='Click me, I will stay visible until you click outside.']")
+    public TextArea text;
```

Comment:
Please do locators static and use it in code without "ClickAwayListenerPage." at the beginning

Reply:
done